tools/untranslated.xml

```diff
@@ -3,11 +3,9 @@
 
     <!-- iOS specific untranslated strings -->
     <string name="import_contacts">Import device contacts</string>
```

Comment:
import_contacts can be deleted as well after the string was added to the android repository - like the line below (import_contacts_message).